src/bin/common/parse_command.ml

```diff
@@ -1325,9 +1325,11 @@ let parse_theory_opt =
     Arg.(value & flag & info ["tighten-vars"] ~docs ~doc) in
 
   let use_fpa =
-    let doc = "Floating-point builtins are always enabled and this option has
-    no effect anymore. It will be removed in a future version." in
-    let deprecated = "this option is always enabled" in
+    let doc =
+      "Enable Floating-point builtins. This option requires to load an fpa \
+       prelude with `--prelude`. It will be removed in a future version."
+    in
+    let deprecated = "Please use the new option `--enable-theories fpa" in
```

Comment:
I think this change is no longer relevant?

Reply:
On the other hand, the warning message when a "fpa-theory" prelude is loaded should be changed to mention `--enable-theory fpa`, I think right now it still says "this is no longer needed".

Reply:
You are right. I thought we still need `use_fpa` but the fpa primitives are now builtins.